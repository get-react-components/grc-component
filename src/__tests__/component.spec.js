import React from 'react';
import renderer from 'react-test-renderer';
import BaseComponent from '../base-component';

test('Rendered BaseComponent', () => {
  const component = renderer.create(
    <div className="base-component">
      <h1>Hello World</h1>
    </div>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});