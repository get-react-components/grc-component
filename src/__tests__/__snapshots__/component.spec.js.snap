// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendered BaseComponent 1`] = `
<div
  className="base-component"
>
  <h1>
    Hello World
  </h1>
</div>
`;
